wiki/Article_styling_criteria/Formatting/ko.md

```diff
@@ -0,0 +1,903 @@
+# 서식
+
+*작성 표준에 대한 내용은 [Article style criteria/Writing](../Writing)를 참조하세요.*\
+*알림: 이 문서에서는 [RFC 2119](https://tools.ietf.org/html/rfc2119)에 따라 요구사항 수준을 설명합니다.*
```

Comment:
the rfc 2119 link is meaningless: the memo is in English and explains how English words are to be used and perceived, while the translation is Korean